proto/expr.proto

```diff
@@ -100,6 +101,8 @@ message ExprNode {
     CHAR_LENGTH = 225;
     REPEAT = 226;
     CONCAT_OP = 227;
+    CONCAT = 286;
+    CONCAT_VARIADIC = 287;
```

Comment:
`concat` is a new function right? (It seems release note isn't complete

Reply:
Yes. Thanks for reminding.